README.md

```diff
@@ -156,9 +156,9 @@ The extra space within a label above the "cap height" and below the "baseline" o
 
 ## Requirements
 
-* iOS 12.0 or later
```

Comment:
Apple no longer supports shipping to iOS 12, so there's no point in supporting it here